config/locales/se.yml

```diff
@@ -0,0 +1,14 @@
+se: 
+  flash[:error]: Fält kan inte vara tomma. Din artikel kunde inte sparas
+  article_successfully_created: Artikel skapad.
+  published_on: Publicerad 
+  Registration: Registrering
```

Comment:
'registration' so everything is with small letters.
We need to change Registration to registration in the views though